cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -23,6 +23,1206 @@ Cask Data Application Platform Release Notes
+  Fixed a problem with the instructions for using Docker.
+
+- `CDAP-2855 <https://issues.cask.co/browse/CDAP-2855>`__ -
+  Fixed a problem with the example builds in VM failing with a Maven dependency error.
+
+- `CDAP-2860 <https://issues.cask.co/browse/CDAP-2860>`__ -
+  Fixed a problem with the documentation for updating dataset properties.
+
+- `CDAP-2861 <https://issues.cask.co/browse/CDAP-2861>`__ -
+  Fixed a problem with CDAP-UI not mentioning required fields in all entry forms.
+
+- `CDAP-2862 <https://issues.cask.co/browse/CDAP-2862>`__ -
+  Fixed a problem with CDAP-UI creating multiple namespaces with the same name.
+
+- `CDAP-2866 <https://issues.cask.co/browse/CDAP-2866>`__ -
+  Fixed a problem with FileBatchSource not reattempting to read in files if there is a failure.
+
+- `CDAP-2870 <https://issues.cask.co/browse/CDAP-2870>`__ -
+  Fixed a problem with Workflow Diagrams.
+
+- `CDAP-2871 <https://issues.cask.co/browse/CDAP-2871>`__ -
+  Fixed a problem with the Cloudera Manager Hbase Gateway dependency.
+
+- `CDAP-2895 <https://issues.cask.co/browse/CDAP-2895>`__ -
+  Fixed a problem with a put operation on the WorkflowToken not throwing an exception.
+
+- `CDAP-2899 <https://issues.cask.co/browse/CDAP-2899>`__ -
+  Fixed a problem with Mapreduce local dirs not getting cleaned up.
+
+- `CDAP-2900 <https://issues.cask.co/browse/CDAP-2900>`__ -
+  Fixed a problem with exposing app.template.dir as a config option.
+
+- `CDAP-2904 <https://issues.cask.co/browse/CDAP-2904>`__ -
+  Fixed a problem with "Make Request" button overlapping with paths when a path is long.
+
+- `CDAP-2912 <https://issues.cask.co/browse/CDAP-2912>`__ -
+  Fixed a problem with HBaseQueueDebugger not sorting queue barriers correctly.
+
+- `CDAP-2922 <https://issues.cask.co/browse/CDAP-2922>`__ -
+  Fixed a problem with datasets created through DynamicDatasetContext not having metrics context.
+  Datasets in MapReduce and Spark programs, and workers, were not emitting metrics.
+
+- `CDAP-2925 <https://issues.cask.co/browse/CDAP-2925>`__ -
+  Fixed a problem with the documentation on how to create datasets with properties.
+
+- `CDAP-2932 <https://issues.cask.co/browse/CDAP-2932>`__ -
+  Fixed a problem with the AdapterClient getRuns method constructing a malformed URL.
+
+- `CDAP-2935 <https://issues.cask.co/browse/CDAP-2935>`__ -
+  Fixed a problem with the logs endpoint to retrieve the latest entry not working correctly.
+
+- `CDAP-2940 <https://issues.cask.co/browse/CDAP-2940>`__ -
+  Fixed a problem with the test case ArtifactStoreTest#testConcurrentSnapshotWrite.
+
+- `CDAP-2941 <https://issues.cask.co/browse/CDAP-2941>`__ -
+  Fixed a problem with the ScriptTransform failing to initialize.
+
+- `CDAP-2942 <https://issues.cask.co/browse/CDAP-2942>`__ -
+  Fixed a problem with the CDAP-UI namespace dropdown failing on standalone restart.
+
+- `CDAP-2945 <https://issues.cask.co/browse/CDAP-2945>`__ -
```

Comment:
I reopened this JIRA, because I believe it is not really resolved.
Remove this line from there?
https://issues.cask.co/browse/CDAP-2945